### DISCUSSION TITLE
IMDSv2 support with openssl 1.0.2j?

### DISCUSSION THREAD

**Discussion by hqho** [aws/aws-sdk-cpp#3080] [Q&A] [answered]
We are trying to transition from using IMDSv1 to IMDSv2, however our OS is still on a very old openssl version 1.0.2j.  Would the newer AWS SDK CPP still work with openssl version 1.0.2j?

**Comment by jmklix** [comment, marked as answer]
Yes, you can use openssl version 1.0.2j with this sdk. Make sure you compile it against the same version that you plan on running it against. That might happen if you are cross compiling or if you have multiple versions of openssl installed
